mappings/net/minecraft/entity/ai/Durations.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_4802 net/minecraft/entity/ai/Durations
+	METHOD method_24505 betweenSeconds (II)Lnet/minecraft/class_4801;
```

Comment:
What does this mean?

Reply:
`Duration.betweenSeconds(1, 2)` returns an int range of `[20, 40]` (ticks) (for choosing a random integer in the interval)